src/braket/aws/aws_quantum_task.py

```diff
@@ -462,7 +462,7 @@ def _(
     *args,
     **kwargs,
 ) -> AwsQuantumTask:
-    if open_qasm_program.inputs is not None:
+    if open_qasm_program.inputs:
```

Comment:
This change is because the to_openqasm method now adds an empty dict instead of None